internal/testdata/doc.go

```diff
@@ -0,0 +1,3 @@
+// package td (TestData) provides literals and utility functions for generating objects-of-concern
+// (objectives, channels, user-agents, etc) for other test packages.
+package td
```

Comment:
Why not call the package `testdata` ? It can be aliased to `td` when imported?

Reply:
Felt like brevity was a virtue here because it will probably be repeatedly invoked in lots of places.

More descriptive name + importers discretion is better though.

Fixed in 5cefca3